feature-configs/cn-ran-overlays/all-in-one/cluster/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  # PTP operator deployment
+  - deployment/ptp
+  # PTP profiles
+  - du-common/ptp
+
```

Comment:
For a CU, we would delete this.  Perhaps we can mark this with a comment.

Reply:
Done